corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -86,16 +86,14 @@ def __init__(self, attrs=None, choices=(), ko_value=None):
         self.choices = list(choices)
 
     def render(self, name, value, attrs=None, choices=()):
-        value = '' if value is None else value
+        self.value = '' if value is None else value
         final_attrs = self.build_attrs(attrs, name=name)
 
         return format_html(
             '<input{0} type="text" data-bind="select2: {1}, {2}">',
             flatatt(final_attrs),
             json.dumps(self._choices_for_binding(choices)),
-            'value: {}'.format(
-                self.ko_value
-            ) if self.ko_value else ""
+            'value: {}'.format(self.ko_value or self.value)
```

Comment:
@esoergel if there is no knockout variable to bind to, it needs to be passed to the `value` attr, otherwise `data-bind` has no meaning....